app/app.py

```diff
@@ -26,6 +27,8 @@ def coconut():
     running_error = False
     proc = None
     python_code = ''
+    coconut_error_lines = []
+    python_error_lines = []
```

Comment:
This feels like this is too many local variables... could we slightly refactor this so `compile_error` and `running_error` can be removed?

Reply:
I think `compile_error` and `running_error` make the current flow make the most sense. Can't really remove them without adding early returns.